src/Orc.Controls/Orc.Controls.Shared/Controls/ColorLegend/Themes/ColorLegend.xaml

```diff
@@ -165,10 +165,12 @@
                                 <RowDefinition Height="*" />
                                 <RowDefinition Height="30"/>
                             </Grid.RowDefinitions>
+                            
                             <Border BorderBrush="LightGray" BorderThickness="0" Background="WhiteSmoke">
-                                <TextBlock FontWeight="Medium" FontSize="14" Margin="4" TextAlignment="Left"
+                                <ContentControl Content="{TemplateBinding Header}"/>
+                                <!--<TextBlock FontWeight="Medium" FontSize="14" Margin="4" TextAlignment="Left"
```

Comment:
If we need to remove stuff, please don't uncomment but actually delete the lines.